ytmusicapi/mixins/search.py

```diff
@@ -233,7 +233,8 @@ def parse_func(contents):
 
     def get_search_suggestions(self,
                                query: str,
-                               detailed_runs=False) -> Union[List[str], List[Dict]]:
+                               detailed_runs=False,
+                               include_history=False) -> Union[List[str], List[Dict]]:
```

Comment:
There's no need to make this an option, is there? We can just make this the default and remove the option

What's more relevant is a flag to determine whether a suggestion originates from the history. Is this discernible with the current solution?